test/unit_test/builtin/Makefile

```
@@ -0,0 +1,97 @@
+clean:
+	rm -rf $(OBJDIR)
+	$(MAKE) -C $(LIBFT_DIR) clean
+
+fclean: clean
+	rm -f $(NAME)
+	$(MAKE) -C $(LIBFT_DIR) fclean
+
+re: fclean all
+
+debug: CCFLAGS += $(DEBUG_FLAGS)
+debug:
```

checkmake (0.3.2)

[warning] Target "debug" defined multiple times (lines 91 and 92).

(uniquetargets)